src/Space.cpp

```diff
@@ -263,7 +263,11 @@ Space::Space(Game *game, RefCountedPtr<Galaxy> galaxy, const Json &jsonObj, doub
 	try {
 		Json bodyArray = spaceObj["bodies"].get<Json::array_t>();
 		for (Uint32 i = 0; i < bodyArray.size(); i++)
+		{
+			if(bodyArray[i]["is_not_in_space"].is_boolean())
```

Comment:
If you are checking for the presence of a key-value pair and not a specific value (i.e. `true` or `false`), prefer `bodyArray[i].count("is_not_in_space") > 0`. The index operator on non-const maps will mutate the JSON object with a new `"is_not_in_space": null` entry (and throw an exception on const maps if the key doesn't exist).

Reply:
Changed this to `count` instead of `is_boolean` like you suggested.